docs/source/examples/managed-jobs.rst

```diff
@@ -452,6 +452,286 @@ To submit the pipeline, the same command :code:`sky jobs launch` is used. The pi
+  (simple-workload, pid=4150) |  GPU   GI   CI              PID   Type   Process name                        GPU Memory |
+  (simple-workload, pid=4150) |        ID   ID                                                               Usage      |
+  (simple-workload, pid=4150) |=========================================================================================|
+  (simple-workload, pid=4150) |  No running processes found                                                             |
+  (simple-workload, pid=4150) +-----------------------------------------------------------------------------------------+
+  ✓ Job finished (status: SUCCEEDED).
+  ✓ Managed job finished: 2 (status: SUCCEEDED).
+
+The job will be launched on one of the available workers in the pool.  
+Currently, each worker is **exclusively occupied** by a single managed job at a time.  
+Support for running multiple jobs concurrently on the same worker will be added in the future.
+
+Submit multiple jobs at once
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pools support a :code:`--num-jobs` flag to conveniently submit multiple jobs at once.  
+Each job will be assigned a unique environment variable :code:`$SKYPILOT_JOB_RANK`, which can be used to determine the job partition.
+
+For example, if you have 1000 prompts to evaluate, each job can process prompts with sequence numbers  
+:code:`$SKYPILOT_JOB_RANK * 100` to :code:`($SKYPILOT_JOB_RANK + 1) * 100`.
+
+Here is a simple example:
+
+.. code-block:: yaml
+
+  # batch-workload.yaml
+  name: batch-workload
+
+  resources:
+    accelerators: {H100:1, H200:1}
+
+  run: |
+    echo "Job rank: $SKYPILOT_JOB_RANK"
+    echo "Processing prompts from $(($SKYPILOT_JOB_RANK * 100)) to $((($SKYPILOT_JOB_RANK + 1) * 100))"
+    # Actual business logic here...
+    echo "Job $SKYPILOT_JOB_RANK finished"
+
+Use the following command to submit them to the pool:
+
+.. code-block:: console
+
+  $ sky jobs launch --pool gpu-pool --num-jobs 10 batch-workload.yaml
+  YAML to run: batch-workload.yaml
+  Submitting to pool 'gpu-pool' with 10 jobs.
+  Managed job 'batch-workload' will be launched on (estimated):
+  Use resources from pool 'gpu-pool': 1x[H200:1, H100:1].
+  Launching 10 managed jobs 'batch-workload'. Proceed? [Y/n]: Y
+  Launching managed job 'batch-workload' (rank: 0) from jobs controller...
+  ...
+  Launching managed job 'batch-workload' (rank: 9) from jobs controller...
+  Jobs submitted with IDs: 3,4,5,6,7,8,9,10,11,12.
+  📋 Useful Commands
+  ├── To stream job logs:                 sky jobs logs <job-id>
+  ├── To stream controller logs:          sky jobs logs --controller <job-id>
+  └── To cancel all jobs on the pool:     sky jobs cancel --pool gpu-pool
+
+All of the jobs will be launched in parallel.  
+Note that the maximum concurrency is limited by the number of workers in the pool.  
+To enable more jobs to run simultaneously, increase the number of workers when creating the pool.
```

Comment:
```suggestion
All jobs will be launched in parallel.  
Note that the maximum concurrency is limited by the number of workers in the pool.  
To enable more jobs to run simultaneously, increase the number of workers when creating the pool.
```

By "All jobs will be launched in parallel.  " do we want to say they will go into PENDING in parallel?

Reply:
maybe we just delete the first line?